config/secrets.yml

```
@@ -1,7 +1,14 @@
 development:
+  primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
+  key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
```

You've still not done the change to make it clear that these are for active_record_encryption. 

I think either set them as a hash:

```suggestion
  active_record_encryption:
    primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
    key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
```

or prefix them both:

```suggestion
  active_record_encryption_primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
  active_record_encryption_key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
```